L1Trigger/L1TMuonEndCap/interface/PtAssignmentEngineAux2017.h

```diff
@@ -29,10 +29,10 @@ class PtAssignmentEngineAux2017 {
   void unpack2bRPC(int rpc_2b, int& rpcA, int& rpcB, int& rpcC) const;
```

Comment:
No changes to this file except spaces ... will probably make the review easier if we just leave it untouched.